src/common/k8s-api/kube-object.ts

```diff
@@ -375,6 +379,12 @@ export type ScopedNamespace<Namespaced extends KubeObjectScope> = (
       : string | undefined
 );
 
+const resourceApplierAnnotationsForFiltering = [
+  "kubectl.kubernetes.io/last-applied-configuration",
```

Comment:
Is this an added functionality?

Reply:
No I moved this from the `src/common/k8s-api/endpoints/resource-applier.api.ts` when I deleted that file.

Reply:
Ok. Resolved.